### PR TITLE
Display namespace and workloads in front-end

### PR DESCRIPTION
**Pull request by viveksinghggits**
1. Get NS from backend
2. Get workloads of NS from backend
3. Display them FE

Just open index.html to test things.

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        <div id="middle" class="mainboxes">
+
+        </div>
+        <div id="right" class="mainboxes">
+
+        </div>
+    </div>
+</body>
+<script type="text/javascript" src="scripts/display_policies.js"></script>
+</html>
```
Comment (PrasadG193) [inline web/index.html]:
```suggestion
</html>

```

File: web/scripts/display_policies.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
+const BASE_URL="http://localhost:8080"
```
Comment (PrasadG193) [inline web/scripts/display_policies.js:1]: can we read this port from env var `KUBERNETS_SERVER_PORT`, just like we are doing in go server? https://github.com/infracloudio/kubernets/blob/master/cmd/serve/server.go#L17

File: web/style/index.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+#left{
+    border-right: 1px solid gray;
+}
+#middle{
+
+}
+#right{
+    border-left: 1px solid gray;
+}
```
Comment (PrasadG193) [inline web/style/index.css]:
```suggestion
}

```

**Review by PrasadG193 — CHANGES_REQUESTED** [review]

File: web/scripts/display_policies.js (hunk 2 above)
Comment (viveksinghggits) [reply web/scripts/display_policies.js:1]: reading env env in vanilla JS is not that easy, we can look into that later.

Comment (PrasadG193) [reply web/scripts/display_policies.js:1]: sounds good

**Review by PrasadG193 — APPROVED** [review]

**Pull request merged**